src/gui/sharelinkwidget.cpp

```diff
@@ -473,6 +485,20 @@ void ShareLinkWidget::slotPushButtonCopyLinkPressed()
 #endif
 }
 
+void ShareLinkWidget::slotPushButtonMailLinkPressed()
+{
+    QString fileName = _sharePath.mid(_sharePath.lastIndexOf('/') + 1);
+
+    QDesktopServices::openUrl(QUrl(QString(
+            "mailto: "
+            "?subject=%1 has shared %2 with you"
```

Comment:
Is there some URL percent encoding stuff needed?  
Maybe not since it's openUrl